cocos/renderer/CCTextureAtlas.cpp

```diff
@@ -612,7 +612,7 @@ void TextureAtlas::drawNumberOfQuads(ssize_t numberOfQuads, ssize_t start)
     if(!numberOfQuads)
         return;
     
-    GL::bindTexture2D(_texture->getName());
```

Comment:
This line will cause OpenGL error with the Label atlas and BMFont rendering with ETC texture.